bacs/src/main/java/com/adyen/checkout/bacs/BacsDirectDebitComponent.kt

```diff
@@ -23,20 +26,24 @@ import kotlinx.coroutines.flow.onEach
  */
 class BacsDirectDebitComponent(
     savedStateHandle: SavedStateHandle,
-    private val bacsDirectDebitDelegate: BacsDirectDebitDelegate,
+    override val delegate: BacsDirectDebitDelegate,
     configuration: BacsDirectDebitConfiguration
 ) : BasePaymentComponent<BacsDirectDebitConfiguration, BacsDirectDebitInputData, BacsDirectDebitOutputData,
-    BacsDirectDebitComponentState>(savedStateHandle, bacsDirectDebitDelegate, configuration) {
+    BacsDirectDebitComponentState>(savedStateHandle, delegate, configuration),
+    ViewProvidingComponent {
 
-    override val inputData: BacsDirectDebitInputData = BacsDirectDebitInputData()
+    override val viewFlow: Flow<ComponentViewType?> get() = delegate.viewFlow
```

Comment:
It's a bit better to not use a custom getter, like this 👇 It prevents some overhead code generation. I always think of it in this way: is the value (in this case `delegate.viewFlow`) subject to change (a `var` or calling a method)? If so, use a custom getter. If not, directly assign the value
```Kotlin
override val viewFlow: Flow<ComponentViewType?> = delegate.viewFlow
```